go-controller/pkg/node/default_node_network_controller.go

```diff
@@ -1304,6 +1304,10 @@ func (nc *DefaultNodeNetworkController) Start(ctx context.Context) error {
 		ovspinning.Run(nc.stopChan)
 	}()
 
+	if err := nc.routeManager.CleanupStaleRoutes(); err != nil {
```

Comment:
as long as route manager is correctly programmed we wont have disruption right? but how do you know it is correctly programmed here by all route manager consumers? e.g. EIP controller for multi NIC.

Reply:
My assumption was by the time we get here all the watchers have started up, so kapi objects have been processed, and now it is safe to run the cleanup. Is that not true?